gravitee-apim-console-webui/package.json

```diff
@@ -20,11 +20,19 @@
     "@asyncapi/web-component": "1.2.26",
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2024-43799   
**Check Name:** send: Code Execution Vulnerability in Send Library   
**Severity:** MEDIUM   
**Fixed Version:** 0.19.0   
**Reachable Path(s) Found:** No  
**Description:** Send is a library for streaming files from the file system as a http response. Send passes untrusted user input to SendStream.redirect() which executes untrusted code. This issue is patched in send 0.19.0.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2024-43799